### PR TITLE
home2

### PR DESCRIPTION
**Pull request by negar-binary**
add a second homepage for AB testing and remove ad pages that aren't used

### CODE REVIEW

**Comment by borisyankov** [comment]
Guys, we should do rebase more often for pull requests like this.
@raunak-binary what do you think?

https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

**Comment by raunak-binary** [comment]
@borisyankov I agree we should keep our pull request up to date with master as it leads to less conflict later

File: src/sass/pages/ad.scss [hunk 1]
```diff
@@ -1,275 +0,0 @@
...
-    display: inline-block;
-}
-.xhead2 {
-    margin-top: 0px;
-    margin-bottom: 20px;
-    padding-bottom: 15px;
-    border-bottom: 1px solid $COLOR_GRAY;
-}
-.xlist1 {
-    background: url(../images/pages/home/xicon-1.png) no-repeat;
```
Comment (raunak-binary) [inline src/sass/pages/ad.scss]: can we remove these images as well if we are not using it

File: src/sass/pages/ad.scss [hunk 2]
```diff
@@ -1,275 +0,0 @@
...
-    padding-left: 50px;
-    min-height: 24px;
-    background-size: 30px;
-}
-.xlist2 {
-    background: url(../images/pages/home/xicon-2.png) no-repeat;
```
Comment (raunak-binary) [inline src/sass/pages/ad.scss]: same

File: src/sass/pages/ad1.scss [hunk 3]
```diff
@@ -1,275 +0,0 @@
-/* HOME PAGE RULES */
-.ad1-content#content {
-  width: 100%;
-}
-.ad1-content {
-   #banner {
-    background: url("../images/pages/ad/map.png") repeat 0 0;
```
Comment (raunak-binary) [inline src/sass/pages/ad1.scss]: same remove images if not required

File: src/sass/pages/ad.scss (hunk 1 above)
Comment (negar-binary) [inline src/sass/pages/ad.scss]: I think this is used in homepage

File: src/sass/pages/ad.scss (hunk 2 above)
Comment (negar-binary) [inline src/sass/pages/ad.scss]: @raunak-binary I think this is also used in home

File: src/sass/pages/ad1.scss (hunk 3 above)
Comment (negar-binary) [inline src/sass/pages/ad1.scss]: @raunak-binary I will remove this.

**Comment by raunakkathuria** [comment]
@negar-binary if images are used its ok, just validate all the images that you  have removed, if they are not used just remove them

**Comment by negar-binary** [comment]
@raunak-binary I removed home2 folder from images/pages as they were from a previous home2 and none of the two pictures there were being used. The rest of the images are used.

**Pull request merged**